Readme.md

```diff
@@ -88,6 +88,16 @@ where `[liquid_domain]` is the value of `liquid.domain` from the `liquid.ini` fi
 
 
 ## Configuration
+### Use Pipenv
+```shell
+sudo apt update
+sudo apt install python3-venv python3-pip git curl unzip
+pip3 install pipenv
+pipenv install -r requirements.txt
+pipenv shell
```

Comment:
You can drop this line since we're keeping the `./liquid` helper script. Don't forget to adjust the next paragraph so it doesn't lie :)